hook/mutate_pod.go

```diff
@@ -161,3 +171,29 @@ func (m podMutator) requestedCapacity(ctx context.Context, pod *corev1.Pod, targ
 	}
 	return total, nil
 }
+
+func (m podMutator) requestedEphemeralCapacity(pod *corev1.Pod) (int64, error) {
+	var total int64
+	sizeAttribute := "topolvm.cybozu.com/size"
+	for _, vol := range pod.Spec.Volumes {
+		if vol.CSI == nil {
+			// We only want to look at CSI volumes
+			continue
+		}
+		if vol.CSI.Driver == topolvm.PluginName {
+			if volSizeStr, ok := vol.CSI.VolumeAttributes[sizeAttribute]; ok {
```

Comment:
After making `topolvm.cybozu.com/size` optional, please update this logic too.

Reply:
Done